x-pack/test/functional/es_archives/rules_scheduled_task_id/rules/mappings.json

```diff
@@ -342,119 +342,3 @@
     }
   }
 }
-
-{
```

Comment:
Moved to `x-pack/test/functional/es_archives/rules_scheduled_task_id/tasks/mappings.json`